daemons/attrd/attrd_commands.c

```diff
@@ -607,7 +607,7 @@ attrd_peer_message(crm_node_t *peer, xmlNode *xml)
         attrd_peer_clear_failure(peer, xml);
 
     } else if (safe_str_eq(op, PCMK__ATTRD_CMD_SYNC_RESPONSE)
-              && safe_str_neq(peer->uname, attrd_cluster->uname)) {
+              && pcmk__str_eq(peer->uname, attrd_cluster->uname, pcmk__str_none) == false) {
```

Comment:
safe_str_neq() is always case-insensitive (and that's actually correct here by chance)

Reply:
Fixed by updating the coccinelle file.